File: libgomp/testsuite/libgomp.oacc-c-c++-common/data-already-2.c
/* { dg-skip-if "" { *-*-* } { "*" } { "-DACC_MEM_SHARED=0" } } */

#include <stdio.h>

int
main (int argc, char *argv[])
{
  int i;

#pragma acc data present_or_copy (i)
  {
    fprintf (stderr, "CheCKpOInT\n");
#pragma acc data copyout (i)
    ++i;
  }

  return 0;
}

/* { dg-output "CheCKpOInT(\n|\r\n|\r).*" } */

File: libgomp/testsuite/libgomp.oacc-fortran/data-already-2.f
! { dg-skip-if "" { *-*-* } { "*" } { "-DACC_MEM_SHARED=0" } }

      IMPLICIT NONE

      INTEGER I

!$ACC DATA PRESENT_OR_COPY (I)
      WRITE(0, *) "CheCKpOInT"
!$ACC DATA COPYOUT (I)
      I = 0
!$ACC END DATA
!$ACC END DATA

      END

! { dg-output "CheCKpOInT(\n|\r\n|\r).*" }
